src/lib/y2caasp/clients/admin_role_dialog.rb

```diff
@@ -45,8 +51,36 @@ def contents
       @content = HSquash(
         MinWidth(50,
           # preselect the servers from the DHCP response
-          Y2Caasp::Widgets::NtpServer.new(dhcp_ntp_servers))
+          Y2Caasp::Widgets::NtpServer.new(ntp_servers))
       )
     end
+
+  private
+
+    #
+    # Propose the NTP servers
+    #
+    # @return [Array<String>] proposed NTP servers, empty if nothing suitable found
+    #
+    def ntp_servers
+      # TODO: use Yast::NtpClient.ntp_conf if configured
+      # to better handle going back
+      servers = dhcp_ntp_servers
+      servers << ntp_fallback if servers.empty? && ntp_fallback
```

Comment:
NP: you can also shorten/simplify the condition if in the line below uses a `servers.compact`, avoiding to call `ntp_fallback` twice.